src/control/zoom/ZoomControl.cpp

```diff
@@ -351,6 +355,16 @@ void ZoomControl::pageSizeChanged(size_t page) {
 }
 
 void ZoomControl::pageSelected(size_t page) {
-    updateZoomPresentationValue(page);
-    updateZoomFitValue(page);
+    // Todo (fabian): page selected should do nothing here, since Zoom Controls, which page is selected.
+    //               This results in a logical loop
```

Comment:
It might be useful to add a link to this PR or the issue it fixes here. E.g.
```c++
// Todo (fabian): page selected should do nothing here, since Zoom Controls, which page is selected.
//               This results in a logical loop. See issue #2770.
```